File: cassandra/jython/delete/cassandra_delete.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	cassandra_delete.py
#					Sep/06/2013
#
# ----------------------------------------------------------------
#
import java
import sys
import string
from java.lang import	System
#
from java.sql import DriverManager
#
sys.path.append ('/var/www/data_base/common/python_common')
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_cassandra_manipulate import cassandra_delete_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")
#
key = sys.argv[1]
print (key)
#
#
str_connect = "jdbc:cassandra://localhost:9160/city"
conn = DriverManager.getConnection (str_connect)

cassandra_delete_proc(conn,key)

conn.close()

print ("*** 終了 ***")
#
# ----------------------------------------------------------------

File: cassandra/jruby/delete/cassandra_delete.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	cassandra_delete.rb
#
#					Sep/06/2013
#
# ---------------------------------------------------------------------
include	Java

import	java.sql.Connection
import	java.sql.DriverManager

load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
load '/var/www/data_base/common/jruby_common/cassandra_manipulate.rb'
# ---------------------------------------------------------------------
puts "*** 開始 ***"

key = ARGV[0]
#
puts key

str_connect = "jdbc:cassandra://localhost:9160/city"
conn = DriverManager.getConnection (str_connect)

cassandra_delete_proc(conn,key)

conn.close()

puts "*** 終了 ***"

# ---------------------------------------------------------------------
